docs/features/job-queue.txt

```diff
@@ -4,63 +4,136 @@
  Queue Management
 ##################
 
-The Determined Queue Management system extends the functionality of scheduler options to offer users
-better functionality in scheduling decisions. The primary role of Queue Management is to provide
-better information about job order, which some schedulers are unable to provide. For example, the
-priority scheduler can provide job ordering information via priority, but the info can be inaccurate
-due to backfilling behavior.
+The Determined Queue Management system extends the functionality of the schedulers to offer you
+better functionality regarding scheduling decisions. It does so by providing the Job Queue, which
+can not only provide better information about job ordering, like what jobs are queued up next, but
+can also allow modification of jobs on the fly.
 
-Queue Management is a new feature and is available to the priority scheduler and the Kubernetes
-priority scheduler. It allows users to observe all submitted jobs and their state. It also allow
-users to modify the position of the jobs in the queue either directly or indirectly.
+Queue Management is a new feature and is available to the fair share scheduler, the priority
+scheduler, and the Kubernetes priority scheduler. To start using it, navigate to the ``Job Queue``
+section of the WebUI or use the ``det job`` family of CLI commands. Queue Management allows you to
+observe all submitted jobs and their states, and provides the ability to modify some configuration
+options, like priority, position in the queue, and resource pool.
+
+************
+ Job States
+************
+
+Jobs in the Job Queue can be in the ``Queued`` or ``Scheduled`` states. A ``Queued`` job has been
+received by Determined but has not been allocated resources yet. Any change made to queued jobs
+should take effect as soon as they have been received by Determined. A job in the ``Scheduled``
+state has already been chosen by the scheduler to start execution and it may have already acquired
+resources or be running at that moment. Once it has started running, a progress bar like the one
+seen in the experiments section should appear. A job that has already started to run will not be
```

Comment:
clarify that this is only the case if the job supports reporting progress. (so the users don't expect this for other job types)

Reply:
I addressed this in my patch